decidim-results/app/views/decidim/results/results/show.html.erb

```diff
@@ -34,6 +37,7 @@
         </div>
       </div>
     </div>
+    <%= render partial: "decidim/shared/share_modal" %>
```

Comment:
Not sure we can do it this way, since depending on the model we want to show different stuff. But it could work like this:

```ruby
social_share_values(
  title: translated_attribute(result.title),
  description: translated_attribute(result.description),
  url: result_url(result.id)
  image: whatever
)
```

And then this helper sets the `content_for` for each key-value pair.